README.md

```diff
@@ -50,20 +50,49 @@ Before running the project, these services must be running and acessible:
 
 Let's examine how to run the project:
 
-### Option 1: [OpenShift](https://www.openshift.com/)
-
-You may use the templates in `openshift/templates/` and upload them to
-Openshift to run the project without any further configuration. The template uses two docker images:
-[`quarck/ruby25-openscap`](https://hub.docker.com/r/quarck/ruby25-openscap/) and [`centos/postgresql-96-centos7`](https://hub.docker.com/r/centos/postgresql-96-centos7/).
+### Option 1: Ephemeral Environment with [Bonfire](https://github.com/RedHatInsights/bonfire/)
 
 #### Prerequisites
 
-* [`ocdeployer`](https://github.com/bsquizz/ocdeployer)
+* [`oc`](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz)
+  ([docs](https://docs.openshift.com/dedicated/4/cli_reference/openshift_cli/getting-started-cli.html))
+* [`bonfire`](https://github.com/RedHatInsights/bonfire/)
+
+#### Setup
+
+1. Log in into the ephemeral cluster
+2. Set up bonfire configuration
+
+    To use local deployement confguration update the `~/.config/bonfire/config.yaml` as follows:
+    ```
+    apps:
+    - name: compliance
+      components:
+        - name: compliance
+          host: local
+          repo: ~/path/to/local/compliance-backend
+          path: clowdapp.yaml
+          parameters:
+            REDIS_SSL: 'false'
```

Comment:
Same comment here. REDIS_SSL is already set to false: https://gitlab.cee.redhat.com/insights-platform/cicd-common/-/blob/master/bonfire_configs/ephemeral_apps.yaml#L35